src/cli/encode.rs

```diff
@@ -96,6 +96,9 @@ impl Default for InputFormat {
 pub enum OutputFormat {
     Single,
     SingleBase64,
+    Stream,
+    // TODO: StreamBase64,
```

Comment:
Do we need to open issues for these?

Reply:
We could, but there is no pressing need for these things. So I think leaving the TODOs here as a way to signal if that feature is added here's the area to add it is okay. But I can also remove the TODOs too.

Reply:
Opened issue:
- https://github.com/stellar/rs-stellar-xdr/issues/388